VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -191,4 +194,27 @@ public static void setEndorsedState(@NonNull TextView target,
         }
 
     }
+
+    /**
+     * Renders hyperlinks in the given html to clickable items when applied to the given TextView.
+     *
+     * @param textView The textView which will render the given html.
+     * @param html     The HTML to apply.
+     */
+    public static void linkifyTextView(@NonNull TextView textView, @NonNull String html) {
+        Spanned spannedHtml = (Spanned) DiscussionTextUtils.parseHtml(html);
+        Object[] urlSpans = spannedHtml.getSpans(0, spannedHtml.length(), URLSpan.class);
+        textView.setAutoLinkMask(Linkify.ALL);
+        textView.setMovementMethod(LinkMovementMethod.getInstance());
+        textView.setText(spannedHtml);
+
+        SpannableString tvText = (SpannableString) textView.getText();
```

Comment:
Maybe rename `tvText` to `viewText` or something? I don't like abbreviations in variable names unless they're easily understandable.